PR for SRVLOGIC-353: Document "Managing persistence" section in the OSL docs. (#85633)
openshift/openshift-docs: serverless-logic/serverless-logic-managing-persistence.adoc

:_mod-docs-content-type: ASSEMBLY
[id="serverless-logic-managing-persistence"]
= Managing workflow persistence
:context: serverless-logic-managing-persistence
include::_attributes/common-attributes.adoc[]

toc::[]

> This section describes how to configure a SonataFlow instance to use persistence and store workflow context in a relational database.

Reviewer: ```suggestion
You can configure a `SonataFlow` instance to use persistence and store workflow context in a relational database.
```

https://redhat-documentation.github.io/supplementary-style-guide/#shortdesc
> Avoid self-referential language, such as "This topic covers…​" or "Use this procedure to…​".

By design, Kubernetes pods are stateless. This behavior can pose challenges for workloads that need to maintain the application state across pod restarts. In the case of {ServerlessLogicProductName}, the workflow context is lost when the pod restarts by default.

> To ensure workflow recovery in such scenarios, you must configure workflow runtime persistence. Use the `SonataFlowPlatform` custom resource (CR) or the `SonataFlow` custom resource (CR) to provide this configuration. The scope of the configuration varies depending on which resource you use.

Reviewer: ```suggestion
To ensure workflow recovery in such scenarios, you must configure workflow runtime persistence. Use the `SonataFlowPlatform` custom resource (CR) or the `SonataFlow` CR to provide this configuration. The scope of the configuration varies depending on which resource you use.
```

Only an acronym is needed after the first occurrence.
